test/README.md

```diff
@@ -13,6 +13,31 @@ The project will create 3 containers:
   - Where anax runs, built initially from openhorizon/anax source, uses local copy of source afterwards unless told otherwise
 And depending on which PATTERN is chosen, a series of workload containers
 
+## QuickStart 
+
+```
+sudo apt update -qq -y && sudo apt install -qq -y ssh git jq curl build-essential docker-compose
+sudo addgroup ${USER} docker
+sudo snap install microk8s --classic --channel=1.18/stable
+wget https://golang.org/dl/go1.14.7.linux-amd64.tar.gz
+sudo tar xzf go1.14.7.linux-amd64.tar.gz -C /usr/local/
+export PATH=$PATH:/usr/local/go/bin
+export GOPATH=${HOME}/go
+mkdir ${GOPATH}
+export ANAX_SOURCE=${HOME}/GIT/anax
```

Comment:
This is specific to how the developer wants to set up their git sandboxes. I dont think it will work for everyone.